src/settings/Settings.js

```diff
@@ -99,6 +99,11 @@ export const SETTINGS = {
         displayName: _td('Disable Emoji suggestions while typing'),
         default: false,
     },
+    "MessageComposerInput.enterSends": {
+        supportedLevels: LEVELS_ACCOUNT_SETTINGS,
+        displayName: _td('Use Enter to send message'),
```

Comment:
Well, I don't think it can be mapped to a boolean flag. This seems to fall squarely into keybinding territory.

Reply:
Well the intent is to only offer a toggle of:
![image](https://user-images.githubusercontent.com/2403652/41425136-bf5772f2-6ff7-11e8-8d23-a03855449a5e.png)

Reply:
which is what other chat apps give you

Reply:
I'd still personally vote for a keybinding UI of some kind given the wide range things that also need keybindings. 

Although if you're dead set on the boolean: Maybe have the wording be "Use Enter to send messages, otherwise CTRL+Enter"?